man/crypttab.xml

```diff
@@ -483,6 +483,14 @@
         option implies <option>tcrypt</option>.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>veracrypt-pim=</option></term>
+
+        <listitem><para>Specifies a custom Personal Iteration Multiplier (PIM)
+        value. Only is effective when <option>tcrypt-veracrypt</option> is 
```

Comment:
"It is only effective when..." also it would be good to add a link to some upstream documentation that talks about this feature